src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonDateTimeTestData.cs

```diff
@@ -232,6 +232,11 @@ public static IEnumerable<object[]> InvalidISO8601Tests()
             // High byte expansion - parsing fails early at 254 characters.
             yield return new object[] { "\"" + new string('\u20AC', 250) + "\"" };
             yield return new object[] { "\"" + new string('\u20AC', 260) + "\"" };
+
+            // Whitespace
```

Comment:
Why was this added, out of curiosity?

Reply:
Just to be consistent. There was a line break between the other "sections" in the method except for that one. I can revert if you want.